pkg/kubelet/config/file_linux_test.go

```diff
@@ -75,7 +73,7 @@ func TestReadPodsFromFileExistAlready(t *testing.T) {
 
 	for _, testCase := range testCases {
 		func() {
-			dirName, err := utiltesting.MkTmpdir("file-test")
+			dirName, err := mkTempDir("file-test")
```

Comment:
Why change this? The methods are basically identical.

Reply:
It is changed according to previous review. I don't have preference.

Reply:
It's fine to depend on client-go/util/testing from here. But I won't block the review on this.